src/routes/userRoutes.js

```diff
@@ -41,6 +43,14 @@ router.put(
   editUser
 );
 
+router.post(
+  '/users/change-password',
+  validateToken,
+  validate(changePasswordValidator),
+  authorizeRequest('changePassword'),
+  changePassword
+);
```

Comment:
Wouldn't this be a PUT method?

Reply:
According to online documentation, it should be a post